converter/dicom-cast/src/Microsoft.Health.DicomCast.Hosting/appsettings.json

```diff
@@ -37,6 +37,9 @@
         "PollInterval": "00:01:00",
         "PollIntervalDuringCatchup": "00:00:00"
     },
+    "DicomValidation": {
```

Comment:
Should this property be in DicomCastWroker and maybe have a name like "IgnoreSyncOfInvalidTagValue"?

Reply:
I could move it there, but would we want it to be a separate configuration class still? Currently the DicomCastWorkerConfiguration is only used in DicomCastWorker, where as the DicomValidationConfiguration is used in multiple other locations, such as the patient and imaging property synchronizers. I don't think we would want to pass the whole DicomCastWorkerConfiguration to those places.

Reply:
Sure. The name is still throwing me off, the names should represent customer scenario, not what the implementation does. So maybe
"DicomCast": {
    "Features": {
        "IgnoreSyncOfInvalidTagValue": false
    }
}

Reply:
Updated to as suggested